app/views/action_page/show.html.erb

```diff
@@ -1,12 +1,16 @@
 <% content_for :head do -%>
   <%= render 'action_page/meta_tags' -%>
-  
+
 <% end -%>
 <div class="top-section">
   <div class="container">
     <div class="row">
       <div>
+        <h5 class="category">Category Placeholder</h5>
         <h1><%= @actionPage.title -%></h1>
+        <div class="image">
+          <img src="<%= @actionPage.background_image.url -%>">
```

Comment:
We should use a helper to render the image here so that the url isn't hard-coded.
```suggestion
          <%= image_tag @actionPage.background_image -%>
```

Reply:
Cool, I'll change that.